arrows/ocv/tests/test_image.cxx

```diff
@@ -159,7 +159,7 @@ void
 run_ocv_conversion_tests( cv::Mat const& img )
 {
   // Convert to a vital image and verify that the properties are correct
-  image const& vimg = ocv::image_container::ocv_to_vital( img );
+  image const& vimg = ocv::image_container::ocv_to_vital( img, ocv::image_container::RGB );
```

Comment:
Do we test anywhere that BGR images work? (I don't think we do? Can we add appropriate tests for that?)

Do we expect the conversion tests to work for the same set of BGR images as RGB images? If so, maybe RGB/BGR should be added to the test parameters... (Obviously it does not need to be totally orthogonal to the number of channels, since RGB/BGR is meaningless in the case of a 1-channel image...)